src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -472,6 +472,10 @@ internal virtual void PrepareSession()
             //
             if (null != Credential && Authentication == WebAuthenticationType.None)
             {
+                if (Uri.Scheme != "https" && !AllowUnencryptedAuthentication)
+                {
+                    WriteWarning(WebCmdletStrings.UnencryptedCredentialWarning);
```

Comment:
Maybe `WriteError()`? It seems we never use `WriteWarning()`.

Reply:
Yup, this is what `ConvertTo-SecureString` does if -AsPlainText is used without -Force:
```
2:8ms> $ss = ConvertTo-SecureString "mypassword" -AsPlainText
ConvertTo-SecureString : The system cannot protect plain text input. To suppress this warning and convert the plain
text to a SecureString, reissue the command specifying the Force parameter. For more information ,type: get-help
ConvertTo-SecureString.
At line:1 char:7
+ $ss = ConvertTo-SecureString "mypassword" -AsPlainText
+       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [ConvertTo-SecureString], ArgumentException
    + FullyQualifiedErrorId : ImportSecureString_ForceRequired,Microsoft.PowerShell.Commands.ConvertToSecureStringComm
   and
```

Reply:
@iSazonov I think an error would be a breaking change here. Recall this piece of code is not part of the new `-Authentication` parameter, but part of the legacy `-Credential` usage. If we want this an error here, I will revert and that can be done in a separate PR. If we are going to break it, the better way, IMO, is to remove the legacy usage and have it be completely replaced with the new usage and removing the legacy `-Credential` usage has some cleanup associated with it.

Reply:
It seems the PR is near to merge. I think we shouldn't block it - please revert and open tracking Issue to discuss "warning", "removing" and "force".

Reply:
Maybe error error with the new -Auth Basic (no -AllowUnen) scenario but not the existing -Credential (no -AllowUnen)?

Reply:
@rkeithhill  that is already part of this PR 😄 the new method is terminating errors when you tery to use non-HTTPS and can only be suppressed with `-AllowUnencryptedAuthentication`. 

This piece of code that this comment thread is on was just to add some more whining to the legacy `-Credential` usage which has always casually let you send credentials over HTTP with out so much as a peep.

@iSazonov #5112 for the this. I have reverted that commit (with the exception of the minor spelling fix I the other tests in this PR)